### PR TITLE
Add Support For Apache Cassandra

### PR DESCRIPTION
**Pull request by shiyqw**
There seems to be some problem with rebase.
I just create a new pull request from a new branch.

### CODE REVIEW

File: src/com/oltpbenchmark/benchmarks/wikipedia/util/WikipediaUtil.java [hunk 1]
```diff
@@ -1,44 +0,0 @@
-/******************************************************************************
```
Comment (apavlo) [inline src/com/oltpbenchmark/benchmarks/wikipedia/util/WikipediaUtil.java]: Why did you remove this file?

File: src/com/oltpbenchmark/benchmarks/ycsb/ddls/ycsb-cassandra-ddl.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS USERTABLE;
+CREATE TABLE USERTABLE (
+  	YCSB_KEY INT PRIMARY KEY,
+    FIELD1 VARCHAR,
```
Comment (apavlo) [inline src/com/oltpbenchmark/benchmarks/ycsb/ddls/ycsb-cassandra-ddl.sql]: Fix formatting

File: src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        </procedure>
+        <procedure name="ReadModifyWriteRecord">
+            <statement name="selectStmt">
+                Select * from USERTABLE where YCSB_KEY=? 
+           </statement>
+        </procedure>
+
+        <procedure name="InsertRecord">
+            <statement name="insertStmt">
+                INSERT INTO USERTABLE (YCSB_KEY, FIELD1, FIELD2, FIELD3, FIELD4, FIELD5, FIELD6, FIELD7, FIELD8, FIELD9, FIELD10) VALUES (?,?,?,?,?,?,?,?,?,?,?)
```
Comment (apavlo) [inline src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml]: Do you really need this?

File: src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    <dialect type="CASSANDRA">
+
+        <procedure name="ScanRecord">
+            <statement name="scanStmt">
+                SELECT * FROM USERTABLE WHERE YCSB_KEY &gt; ? AND YCSB_KEY &lt; ? ALLOW FILTERING
+            </statement>
+        </procedure>
+        <procedure name="ReadModifyWriteRecord">
+            <statement name="selectStmt">
+                Select * from USERTABLE where YCSB_KEY=?
```
Comment (apavlo) [inline src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml]: Do you really need this? Please use consistent captialization

File: src/com/oltpbenchmark/benchmarks/wikipedia/util/WikipediaUtil.java (hunk 1 above)
Comment (shiyqw) [reply src/com/oltpbenchmark/benchmarks/wikipedia/util/WikipediaUtil.java]: Added back

File: src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml (hunk 4 above)
Comment (shiyqw) [reply src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml]: I think we do not need this. Removed.

File: src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml (hunk 3 above)
Comment (shiyqw) [reply src/com/oltpbenchmark/benchmarks/ycsb/dialects/cassandra-dialects.xml]: I think we need this because the original command did not include column names

**Comment by apavlo** [comment]
@shiyqw This is still messed up. I am not sure what you are doing with WikipediaUtil? Can you remove that from your history?

**Comment by shiyqw** [comment]
Have created a new PR

**Comment by apavlo** [comment]
Replaced by #228
